pillar/base/haproxy.sls

```diff
@@ -17,6 +17,7 @@ haproxy:
         - docs.python.org
         - doc.python.org
       check: "HEAD /_check HTTP/1.1\\r\\nHost:\\ docs.python.org"
+      rate_limit: 100
```

Comment:
While useful for demonstration, we can probably remove this, since as is we don't see much rate limit concern with this host and it can be added back trivially!